openslides/mediafiles/models.py

```diff
@@ -41,6 +44,7 @@ class Meta:
         default_permissions = ()
         permissions = (
             ('can_see', 'Can see the list of files'),
+            ('can_see_private', 'Can see privately listed files'),
```

Comment:
Please add the permission string to users/site.js to make the string translateable:
https://github.com/OpenSlides/OpenSlides/blob/master/openslides/users/static/js/users/site.js#L986